shell/browser/electron_browser_main_parts.cc

```diff
@@ -371,6 +371,11 @@ void ElectronBrowserMainParts::ToolkitInitialized() {
     electron::UninitializeElectron_gtk();
   }
 
+  electron::InitializeElectron_gdk_pixbuf(gtk::GetLibGdkPixbuf());
+  if (!electron::IsElectron_gdk_pixbufInitialized()) {
+    electron::UninitializeElectron_gdk_pixbuf();
+  }
```

Comment:
This should be a hard check unlike the previous gtk symbols loaded there is no fallback for the gdk_pixbuf calls. Something like the following,

```suggestion
  CHECK(electron::IsElectron_gdk_pixbufInitialized()) << "Fail to load libgdk_pixbuf-2.0.so.0";
```